### ISSUE TITLE
There is ES6 syntax in the dist files in version 2.7.0

### ISSUE THREAD

**Issue by allanwjm** [TargetProcess/tauCharts#570]
You are using **ES6** in the **dist**! Checkout the [CDN link](https://cdn.jsdelivr.net/npm/taucharts@2/dist/taucharts.min.js) on the tauCharts homepage and there are a lot of `=>`, `const`, blabla... in it.

```
ERROR in 1.bundle.js from UglifyJs
Unexpected token: operator (>)
```

That caused webpack build errors for me. If you see this, you might need to use `taucharts@2.6.x`.  I reckon that will also be a problem in browsers.

I'm not sure if it is a bug or **feature**, just FYI.

**Comment by Mavrin** [comment]
Thank you! It is bug.

**Comment by Mavrin** [comment]
Fixed in 2.7.1
